java/README.md

```diff
@@ -55,3 +55,23 @@
      human-readable.
 
 9. Update the code to insert a document with the regular client. What happens?
+
+## KMIP
+
+**KMIP support in this guide is to be considered "best effort"**.
```

Comment:
I suggest clarifying that "best effort" means unstable dependencies. The PR description seems clearer:

> I used the term "best effort" in the README to signify that the KMIP demonstration in the guide uses unsupported and unstable tools (drivers-evergreen-tools and its pykmip implementation).

Perhaps this?

```suggestion
**KMIP support in this guide relies on unstable dependencies**.
```

Reply:
Agree this is clearer, made the update!